permissions/component-jenkins-test-harness-htmlunit.yml

```diff
@@ -5,6 +5,4 @@ cd:
   enabled: true
 paths:
   - "org/jenkins-ci/main/jenkins-test-harness-htmlunit"
-developers:
-  - "drulli"
-  - "@core"
+developers: []
```

Comment:
Fully switched to CD.

Reply:
Btw A plugin with 0 developers listed is considered up for adoption

Reply:
This is a library, not a plugin. You can't find it on plugins.jenkins.io, but we can add Ulli back, if needed.